src/step-input.scss

```diff
@@ -0,0 +1,153 @@
+@import "./settings";
+@import "./new-settings";
+@import "./mixins";
+@import "./functions";
+@import "./button";
```

Comment:
Please don't import the button. This file is huge! You don't need `@include buttonContainerHover();`. Even if you need a mixin like this it's better to copy the mixin instead of importing a big file like button